ember-cli-build.js

```diff
@@ -37,12 +37,27 @@ function resolveRelativeModulePath(name, child) {
 resolveRelativeModulePath.baseDir = () => __dirname;
 
 function toAMD(tree) {
+  const isProduction = process.env.EMBER_ENV === 'production';
+  const isDebug = !isProduction;
+
   return new Babel(tree, {
     moduleIds: true,
     getModuleId: getRelativeModulePath,
     plugins: [
       ['module-resolver', { resolvePath: resolveRelativeModulePath }],
       ['@babel/plugin-transform-modules-amd', { noInterop: true }],
+      [
+        'babel-plugin-debug-macros',
+        {
+          flags: [
+            {
+              source: '@glimmer/env',
+              flags: { DEBUG: isDebug, CI: !!process.env.CI },
+            },
+          ],
+        },
+        '@glimmer/env inlining',
+      ],
```

Comment:
FWIW, this only affects the local tests (not what consumers use). Ember's build pipeline (as well as general Ember apps) uses nearly the same thing...